src/Parsing.hs

```diff
@@ -412,25 +412,8 @@ symbol = do
             Nothing
         )
     else pure $ case last segments of
-      "defn" -> XObj (Defn Nothing) i Nothing
```

Comment:
Shouldn't the definitions for all these `Obj`:s be removed also?

Reply:
No, since the symbols now evaluate to the values shown here, instead of being resolved at parsing. Basically, we just push this behavior back into the expander/evaluator.

Reply:
Ah, OK – now I get it!